tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -119,7 +119,8 @@ q isa MultivariateDistribution
 This means that we can call `rand` to sample from the variational posterior `q`
 
 ```julia
-rand(q)
+using Plots
+Plots.histogram(rand(q, 1_000)[1, :])
```

Comment:
Do you know why this is `Plots.` specification is necessary? We should probably figure out why this is needed before we include it, since Plots should be exporting this method.

Reply:
yes, it doesn't seem to be necessary to explicitly include the `Plots.` prefix; however, in a previous commit, buildkite failed with `ERROR: LoadError: UndefVarError: Plots not defined` (see [here](https://buildkite.com/julialang/turingtutorials/builds/327#01850879-f563-4a6f-8f47-3de0f60ccb3e)). So I explicitly added `Using Plots` and `Plots.histogram` in each use. 
I am investigating why this is needed.